std/os/windows.zig

```diff
@@ -779,3 +779,18 @@ pub fn unexpectedError(err: DWORD) std.os.UnexpectedError {
     }
     return error.Unexpected;
 }
+
+pub fn getSubSystem() ?builtin.SubSystem {
```

Comment:
What do you think about making this be a `pub const subsystem = blk: { ... };` rather than a function? This way it will always be comptime known.

Reply:
sure, but this implementation isn't neccessarily complete. if you have `const wWinMain = 0` in your root file it will report the wrong susbystem.